components/tests/ui/testcases/web/create_scenario.txt

```diff
@@ -0,0 +1,303 @@
+    ${projectId}=                           Run Keyword If                          '${isOrphaned}' == '${notOrphaned}'  Get Text            css=tr.data_heading_id strong
+    ${datasetId}=                           Create Dataset                          ${CtxDatasetName}
+    Run Keyword If                          '${isOrphaned}' == '${orphaned}'        Dataset Exists and Is Orphaned      ${datasetId}
+    Run Keyword If                          '${isOrphaned}' == '${notOrphaned}'     Dataset Exists and Is Not Orphaned  ${datasetId}        ${projectId}
+    [Return]                                ${datasetId} 
+
+Create Dataset Using Right Click and Check If Orphaned
+    [Arguments]                             ${RootId}       ${DatasetName}      ${isOrphaned}
+    ${projectId}=                           Run Keyword If                          '${isOrphaned}' == '${notOrphaned}'  Get Text            css=tr.data_heading_id strong
+    Right Click Create P/D/S                ${RootId}       Dataset                 ${DatasetName}
+    ${datasetId}=                           Check Right And Center Panels For Active Container                          Dataset             ${DatasetName}
+    Run Keyword If                          '${isOrphaned}' == '${orphaned}'        Dataset Exists and Is Orphaned      ${datasetId}
+    Run Keyword If                          '${isOrphaned}' == '${notOrphaned}'     Dataset Exists and Is Not Orphaned  ${datasetId}        ${projectId}
+    [Return]                                ${datasetId}  
+
+Dataset Exists and Is Orphaned
+    [Arguments]                             ${datasetId}    
+    ${nodeId}=                              Wait for Dataset Node               ${datasetId}
+    Wait Until Page Contains Element        xpath=//li[@id='${treeRootId}']/ul/li[@id='${nodeId}']
+
+Dataset Exists and Is Not Orphaned
+    [Arguments]                             ${datasetId}                        ${projectId}
+    ${datasetnodeId}=                       Wait for Dataset Node               ${datasetId}
+    ${projectnodeId}=                       Wait for Project Node               ${projectId}
+    Wait Until Page Contains Element        xpath=//li[@id='${treeRootId}']/ul/li[@id='${projectnodeId}']/ul/li[@id='${datasetnodeId}']
+
+Dataset Should Contain Image 
+    [Arguments]                             ${imageId}      ${datasetId}
+    [Documentation]                         Checks if the newly created dataset contains the image. Very Specific test, needs to be generalised.
+    Wait Until Page Contains Element        css=tr.data_heading_id strong
+    Select Dataset By Id                    ${datasetId}
+    ${nodeId}=                              Wait For Dataset Node               ${datasetId}
+
+    Wait Until Element Is Visible           xpath=//li[@id='${nodeId}']/ul/li[1]/a
+    Click Element                           xpath=//li[@id='${nodeId}']/ul/li[1]/a
```

Comment:
This assumes that the Image is the first image in the Dataset. While this works for this test, it would be nicer to have it more generalised. Probably all you need to do is to click on the Dataset, then wait until image appears as a child element under the Dataset node in the tree. Shouldn't really need to load right panel etc. Once you've updated this, you could also update the [Documentation] for this above.